packages/dashboard/src/pages/Dash/PlatformSizes.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import cx from 'classnames';
+import byteSize from 'byte-size';
+import { Stats } from '../../types';
+
+interface Props {
+  stats: Stats;
+}
+
+const MB_AS_BYTES = 1024 * 1024;
+
+export function PlatformSizes({ stats }: Props) {
+  const publicAssets = React.useMemo(
+    () =>
+      stats.assets.filter(
+        (asset) =>
+          asset.type === 'asset' &&
+          !/\.(js)?bundle\.(map|json)$/.test(asset.name)
+      ),
+    [stats.assets]
+  );
+
+  const total = React.useMemo(
+    () => publicAssets.reduce((acc, asset) => acc + asset.size, 0),
+    [publicAssets]
+  );
+
+  const modules = React.useMemo(
+    () =>
+      publicAssets
+        .filter((asset) => /\.((js)?bundle|js)$/.test(asset.name))
+        .reduce((acc, asset) => acc + asset.size, 0),
+    [publicAssets]
+  );
+
+  const getColor = React.useCallback((size: number) => {
+    return cx({
+      'text-green-500': size < 10 * MB_AS_BYTES,
+      'text-yellow-500': size < 20 * MB_AS_BYTES,
```

Comment:
Will it work properly? For size 9MB it will add `green` and `yellow` simultaneously